explore/src/main/scala/explore/observationtree/ObsList.scala

```diff
@@ -73,14 +73,13 @@ object ObsList:
 
   private def scrollIfNeeded(targetObs: Observation.Id) =
     Callback {
-      // 'getElementById' does not return any optional type, so coerce to js.UndefOr
-      val target: js.UndefOr[Element] =
-        dom.document.getElementById(s"obs-list-${targetObs.toString}")
-      target.foreach { t =>
-        val rect = t.getBoundingClientRect()
-        if (rect.top < 0) t.scrollIntoView()
-        if (rect.bottom > dom.window.innerHeight) t.scrollIntoView(false)
-      }
+      Option(dom.document.getElementById(s"obs-list-${targetObs.toString}"))
```

Comment:
This got me thinking, are refs the proper react way to do this?

Reply:
Potentially, but with a list I don't think that would work. Or you'd have to have a single ref that you set on the last added observation, but that seems a lot more complex than a query selector

Reply:
Right, now I recall I tried that